.github/actions/sourceclear/entrypoint.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh -l
+echo "Download srcclr ci script ..."
+curl -sSL https://download.sourceclear.com/ci.sh --output srcclr.sh
+echo "Script saved to srcclr.sh"
+chmod a+x srcclr.sh
+echo "Script execution rights added"
+
+# monorepo
+echo "Starting scan on . (root) ..."
```

Comment:
Is it not possible to make a loop on every subfolder of `./packages` to scan them? It could avoid to have to add each package one by one.

Reply:
I can change that to have a list of folder to scan, yes, but not all the folders. There are some of them that we don't need to scan, like generator.

Reply:
Oh ok, i was thinking that we wanted to scan all of them. My comment was only about to avoid to have to maintain a list of packages 😅 
So you can keep it if you want :D, good for me :)

Reply:
It will be easier to have a simple list instead of searching into the code. I'll do the change